allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -118,6 +119,35 @@ def get_cell_specimen_ids(self):
             raise ValueError(f'cell_specimen_id values not assigned for {self.ophys_experiment_id}')
         return cell_specimen_ids
 
+    def deserialize_image(self, sitk_image):
+        '''
+        Convert SimpleITK image returned by the api to an Image class:
+
+        Args:
+            sitk_image (SimpleITK image): image object returned by the api
+
+        Returns
+            img (allensdk.brain_observatory.behavior.image_api.Image):
+                Image class with the following attributes:
+                data : np.ndarray
+                    Image data points
+                spacing : tuple
+                    Spacing describes the physical size of each pixel
+                unit : str
+                    Physical unit of the spacing (currently constrained to be isotropic)
+        '''
+        img = ImageApi.deserialize(sitk_image)
+        return img
+
+    def get_max_projection(self):
```

Comment:
docstring describing this image and specifying its data type, like:
```
""" Returns an image whose values are the maximum obtained values at each pixel of the ophys movie over time.'

Returns
----------
allensdk.brain_observatory.behavior.image_api.Image : 
    array-like interface to max projection image data and metadata
"""
```

Reply:
Resolved by 52f99a